tests/cjs-path-mapping/tsconfig-base.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Do you want to write those tests?

Reply:
I have to get myself familiar with the `test` setup. Currently, I'm not sure I could make a test setup which `tsconfig` contains 

**tsconfig.node12.json**
```json
{
  "ts-node": {
    "esm": true <-- can this be done by the current test infra?
  }
  "compilerOptions": {
    "baseUrl": "./",
    "paths": [ ... ],
    "moduleResolution": "node12"
  }
}
```
**package.json**
```json
{
  "devDependencies": {
    "typescript" "^10.6.0"
}

Reply:
> "experimentalPathMapping": "both"

This is sufficient; we do not need to test any other variations of this option.  Before merging this feature, I will probably merge `"experimentalPathMapping"` into [`"experimentalResolverFeatures"`](https://typestrong.org/ts-node/docs/options#experimentalresolverfeatures)  This will simplify things and set us up to enable by default in the future, once our various resolver behaviors stabilize.

> which version of TypeScript ts-node uses internally?

We use Typescript as a peer dependency.  We use the version in your project, supporting as far back as 2.7.  Our test matrix on Github Actions runs multiple versions; check the Github Actions configuration for details.  You shouldn't need to care for these tests, since as I mentioned previously, we can completely disable typechecking for these tests.  (discussion: https://github.com/TypeStrong/ts-node/pull/1664#discussion_r816426376)